components/dashboard/src/data/organizations/orgs-query.ts

```diff
@@ -61,14 +61,31 @@ export function useCurrentOrg(): { data?: Organization; isLoading: boolean } {
         return { data: undefined, isLoading: true };
     }
     let orgId = localStorage.getItem("active-org");
+    let org: Organization | undefined = undefined;
```

Comment:
Only kind of, because before, there was no way of specifying the org ID in the URL search. This means that the only two possible sources for getting an org: 1) from `localStorage` and then, only if there was no value present, from the API response (`orgs.data[0]`).

If we wanted to keep the order, we'll have to change the code from checking `if (orgSlugParam` or `if (orgIdParam)` to `if (orgSlugParam && !org)` to not override if we already got a value.

Reply:
This is a dud, sorry! We resolved this in sync, but the order is proper and I was just projecting a bug I found onto it.

Reply:
The bug I was thinking of was fixed in [`1515e72` (#20431)](https://github.com/gitpod-io/gitpod/pull/20431/commits/1515e72a41007a32cda0cf8c6cef6c3379be809a)